rails-tasklist/app/controllers/tasks_controller.rb

```diff
@@ -25,9 +25,39 @@ def delete
     redirect_to "/"
   end
 
+  def update
+    id = params[:id]
+    task = Task.find(id)
+    task.update(
+      name: task_params[:task][:name],
+      description: task_params[:task][:description]
+      )
+    # task.update(task_params[:task])
+    redirect_to "/"
+  end
+
+  def completed
+    id = params[:id]
+    task = Task.find(id)
+    if task.completed_at.nil?
+      task.update(completed_at: Time.now)
+    else
+      task.update(completed_at: nil)
```

Comment:
Is this a "toggle" kind of behavior where if a task is completed it can be "uncompleted"?